src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final Random RANDOM = new Random();
+
+    public String getRandomColor() {
+        int indexRandom = RANDOM.nextInt(Color.values().length);
+        String randomColor = Color.values()[indexRandom].name();
+
+        return switch (randomColor) {
+            case "RED" -> "red";
+            case "GREEN" -> "green";
+            case "BLUE" -> "blue";
+            case "WHITE" -> "white";
+            case "BLACK" -> "black";
+            default -> throw new IllegalStateException("Unexpected value: " + randomColor);
+        };
+    }
```

Comment:
The switch statement in the 'getRandomColor' method is redundant since you're converting the enum names to lowercase strings, which can be done directly using 'toLowerCase()' method on the enum's name. This will simplify the method significantly.

Reply:
agree with comment